api/openapi-spec/swagger.json

```diff
@@ -45699,7 +45699,44 @@
     }
    },
    "io.k8s.apimachinery.pkg.apis.meta.v1.Patch": {
-    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body."
+    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body.",
+    "type": "array",
+    "items": {
+       "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.PatchDocument"
+      }
+   },
+    "io.k8s.apimachinery.pkg.apis.meta.v1.PatchDocument": {
+    "description": "A JSONPatch document as defined by RFC 6902.",
```

Comment:
I see merge patch (https://tools.ietf.org/html/rfc7386) support in a few places, which other patch type is in use?